core/spec.md

```diff
@@ -2053,45 +2082,86 @@ Regardless of the mechanism used to update the capabilities, the Registry's
 
 In order for a client to discover the list of available values for each
 capability, an HTTP `GET` MAY be sent to the `/capabilities` API with the
-`?offered` query parameter and the response MUST adhere to the following:
+`?offered` query parameter and the response MUST adhere to the following
+(which borrows many of the same structure from the `/model` API):
 
 ```yaml
 GET /capabilities?offered
 
 {
-  "STRING": [ OFFERED-VALUES ] *
+  "STRING": {
+    "type": "TYPE",
+    "item": {
+      "type": "TYPE"
+    }, ?
+    "enum": [ VALUE, * ], ?
+    "min": VALUE, ?
+    "max": VALUE, ?
+    "documentation": "URL" ?
+  }, *
 }
 ```
 
 Where:
-- `"STRING"` is the capability name.
-- `OFFERED-VALUES` is an array of the valid values for that capability. The
-  data type of each value MUST adhere to the data type associated with the
-  capability, with the exception of the `#` (range)  variants described below.
+- `STRING` MUST be the capability name.
+- `TYPE` MUST be one of `boolean`, `string`, `integer`, `decimal`, `uinteger`,
+  `array` as defined in [Attributes and
+  Extensions](#attributes-and-extensions).
+- `"item.type"` MUST be one of `boolean`, `string`, `integer`, `decimal`,
+  `uinteger` when `"type"` is `array`, otherwise `"item"` MUST be absent.
```

Comment:
```suggestion
- When `"type"` is `array`, `"item.type"` MUST be one of `boolean`, `string`, `integer`, `decimal`,
  `uinteger`, otherwise `"item"` MUST be absent.
```

Otherwise the scope of the "when" is ambiguous - I originally read it with mental brackets of:

> `boolean`, `string`, `integer`, `decimal`, (`uinteger` when `"type"` is `array`)

... thinking that a correlation between `uinteger` and arrays would be somewhat natural, in terms of array indexes.

Reply:
done